targets/TARGET_NXP/TARGET_LPC176X/TARGET_UBLOX_C027/UBLOX_onboard_modem_api.h

```diff
@@ -0,0 +1,53 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2020 ARM Limited
```

Comment:
please add SPDX id to new files

Reply:
Do you want me to this line?
`SPDX-License-Identifier: Apache-2.0`

Reply:
yes please

Reply:
We already have a job in Travis checking these SPDX, see below: travis-ci/licence_check — Needs review, 4 license issues found (4 files found with license issue) in this PR